lib/tether/tether.go

```diff
@@ -893,3 +913,27 @@ func killHelper(session *SessionConfig) error {
 
 	return nil
 }
+
+// Create necessary folders/files as the src/target for bind mount.
+// See https://github.com/vmware/vic/issues/489
+func createBindSrcTgt() error {
+	for dirPath, dmode := range dirForMinOS {
+		if err := os.MkdirAll(dirPath, dmode); err != nil {
+			return fmt.Errorf("failed to create directory %s: %s", dirPath, err)
+		}
+	}
+
+	// The directory has to exist before creating the new file
```

Comment:
Why don't we ensure that is the case by creating the file's parent directory here, and not having a separate map for the dirs?